src/index.js

```diff
@@ -913,15 +942,20 @@ module.exports = {
 		writeCorsHeaders(route, req, res, isPreFlight) {
 			if (!route.cors) return;
 
+			const origin = req.headers["origin"];
+
+			if (!origin) {
+				throw new NotFoundError(ERR_ORIGIN_NOT_FOUND);
```

Comment:
We need this error if no `Origin`? If I make a simple request in the browser with axios, it is not exist in request header.

Reply:
Hey @icebob . What do you mean by a simple request from the browser? If you don't have the origin set in your header and have the cors enabled, it means the browser cannot validate the request.

Reply:
Please see it. I'm trying to update conduit example. CORS is enabled in API GW settings. I open it in the browser and API GW throw "NotFoundError"s which comes from this line:
![image](https://user-images.githubusercontent.com/306521/41796636-9f603a80-7667-11e8-9e4d-dc1c4ac9b39d.png)

Reply:
Maybe because I call it from "localhost"?

Reply:
Sorry for the delay. I was on vacations. I guess it could skip if no origin but it's weird it's not setting origin if you're testing in a browser.

Reply:
No problem. I figured out that, the browser sends it only if the request host is different than the original host. So if you open a site from `localhost:3000` and send a request to the same host, it won't set `Origin` to the header. But if you send request to e.g. `localhost:4000`, it will set `Origin: http://localhost:3000`. Bearing in mind that, I changed it to skip CORS checking if no `Origin`.